src/renderable/animationsheet.js

```diff
@@ -19,11 +19,12 @@
      * @param {Number} [settings.framewidth] Width of a single frame within the spritesheet
      * @param {Number} [settings.frameheight] Height of a single frame within the spritesheet
```

Comment:
I'm familiar. ;)

Imagine how you might have to modify these lines of code to properly run within `me.Sprite.draw`:

```javascript
// in this case, the entity's anchor point is in relation to the body
// draw the renderable's anchorPoint at the Entity's anchor point
x = ~~( x + (this.anchorPoint.x * this.body.width) - (this.renderable.anchorPoint.x * this.renderable.width));
y = ~~( y + (this.anchorPoint.y * this.body.height) - (this.renderable.anchorPoint.y * this.renderable.height));
```

Copied directly from your patch, above. :) The issue here is that correctly positioning the sprite relies on the sprite anchorPoint *and* the entity anchorPoint. Since the entity is already doing this translation, it makes sense that the debugPanel should do it, as well. After all, the debugPanel is just patching `me.Entity.draw` ... Does it make sense?